packages/core/src/server/socketServer.ts

```diff
@@ -22,20 +22,34 @@ function isEqualSet(a: Set<string>, b: Set<string>): boolean {
   return true;
 }
 
+const parseUrlParams = (url?: string) => {
```

Comment:
Can we use the Node.js built-in query-string module?

https://nodejs.org/api/querystring.html